frontend/src/__tests__/cypress/cypress/pages/acceleratorProfile.ts

```diff
@@ -0,0 +1,240 @@
+import { Modal } from './components/Modal';
+import { TableRow } from './components/table';
+
+class AcceleratorRow extends TableRow {
+  findDescription() {
+    return cy.findByTestId('description');
```

Comment:
Scope selector to the current context.
```suggestion
    return this.find().findByTestId('description');
```